test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -1297,6 +1297,67 @@ public void ControllerView_UsesControllerViewEngine()
             Assert.Same(viewEngine, result.ViewEngine);
         }
 
+        [Fact]
+        public void TryValidateModel_NullModel()
+        {
+            // Arrange
+            var controller = new Controller();
+
+            // Act & Assert
+            Assert.Throws<ArgumentNullException>(
+                 () => controller.TryValidateModel(null));
+        }
+
+
+        [Fact]
+        public void TryValidateModelWithValidModel()
```

Comment:
WOuld be useful to express the expectation as part of the test (e.g. `TryValidateModel_WithValidModel_ReturnsTrue`)